.github/workflows/biotools-pr-validator.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'validate bio.tools json and auto-merge'
+
+on:
+  pull_request:
```

Comment:
is there a way to only trigger this if a file on data/*/*.biotools.json has been modified?